stellar/operations.py

```diff
@@ -36,14 +36,9 @@ def copy_database(raw_conn, from_database, to_database):
     terminate_database_connections(raw_conn, from_database)
 
     if raw_conn.engine.dialect.name == 'postgresql':
-        raw_conn.execute(
-            '''
-                CREATE DATABASE "%s" WITH TEMPLATE "%s";
-            ''' %
-            (
-                to_database,
-                from_database
-            )
+        sqlalchemy_utils.functions.create_database(
+            '%s%s' % (raw_conn.engine.url, to_database),
+            template=from_database
```

Comment:
@pypingou , it was 3 years ago, but from your requirements.txt it looks like you do rely on sqlalchemy-utils and operations.py imports it so it looks like the idea contained in this pull request could be merged.